solidity/ecdsa/contracts/test/upgrades/WalletRegistryV2.sol

```diff
@@ -0,0 +1,1174 @@
+// This is a contract used to test WalletRegistry upgradeability. It is a copy
+// of WalletRegistry contract with some differences marked with `TEST:` comments.
```

Comment:
Non-critical but there were some changes in `WallegRegistry` along the way:
- `authorizationParameters()` function is missing
- `authorizationDecreaseDelay()` is still here but was removed in the original implementation
- `updateAuthorizationParameters` is missing two new parameters
- `AuthorizationParametersUpdated` event is missing a parameter
- `_dkgResultSubmissionGas` field doc lost one line for some reason

Would be great to sync them again at the end.

Same comment to two other copies of `WalletRegistry.sol`.

Reply:
52b0dd92ddda423c56c3ebddebabba913499ea98

Reply:
52b0dd92ddda423c56c3ebddebabba913499ea98